lms/static/sass/application.scss.mako

```diff
@@ -51,6 +51,7 @@
 @import 'shared/modal';
 @import 'shared/activation_messages';
 @import 'shared/unsubscribe';
+@import "views/account-settings";
```

Comment:
This should have its own section outside of the //shared - course area I think.  I would make a "// views - platform" section underneath shared - course 

Additionally, this import might make sense to add to application-extend2.scss.mako since that's where the other platform level views are imported.